app/controllers/qa_controller.rb

```diff
@@ -13,7 +13,8 @@ def show
 private
 
   def qa_config
-    @qa_config ||= YAML.load_file("lib/#{request.path.tr('-', '_')}.yaml")
+    qa_config_path = Rails.application.config.qa_config_path || "lib"
```

Comment:
What about making this method public instead, and then stubbing it in the tests? Or adding `config.qa_config_path` to all of the environments?

Reply:
Turns out a private method can also be stubbed, so I've stubbed it and removed the config.